### DISCUSSION TITLE
How to scroll with mouse wheel when cursor is on scroll bar?

### DISCUSSION THREAD

**Discussion by jtcyacomb** [AvaloniaUI/Avalonia#16369] [Q&A] [answered]
Is there a way to enable scrolling a `ScrollViewer` with the mouse wheel when the mouse cursor is on the `ScrollBar` of the `ScrollViewer`?

For example, if the mouse cursor is on the vertical `ScrollBar` of a `ScrollViewer`, and the mouse wheel is scrolled up or down, scrolling does not occur, currently.

Was this a deliberate decision by the Avalonia team?

**Comment by maxkatz6** [comment]
Sounds like a bug to me.

**Reply by maxkatz6** [reply]
Can you open an issue if you can reproduce it on latest builds?

**Reply by jtcyacomb** [reply, marked as answer]
Thanks for your reply. I have opened an issue: #16372
